web/src/components/OpeningHours.tsx

```diff
@@ -0,0 +1,89 @@
+import moment from 'moment'
+import React, { ReactElement } from 'react'
+import { useTranslation } from 'react-i18next'
+import styled from 'styled-components'
+
+import { OpeningHoursModel } from 'api-client'
+import { UiDirectionType } from 'translations/src'
+
+import Collapsible from './Collapsible'
+import OpeningEntry from './OpeningEntry'
+
+const OpeningLabel = styled.span<{ isOpened: boolean }>`
+  color: ${props => (props.isOpened ? props.theme.colors.positiveHighlight : props.theme.colors.negativeHighlight)};
+  margin: 0 auto;
+`
+
+const Content = styled.div`
+  padding: 8px;
+  font-size: clamp(0.55rem, 1.6vh, 0.85rem);
+`
+
+const TitleContainer = styled.div`
+  display: flex;
+  flex: 1;
+  font-weight: 700;
+  font-size: clamp(0.55rem, 1.6vh, ${props => props.theme.fonts.hintFontSize});
+`
+
+type OpeningHoursProps = {
+  isCurrentlyOpened: boolean
+  direction: UiDirectionType
+  openingHours: OpeningHoursModel[] | null
+  isTemporarilyClosed: boolean
+}
+
+const getOpeningLabel = (isTemporarilyClosed: boolean, isCurrentlyOpened: boolean): string => {
+  if (isTemporarilyClosed) {
+    return 'openingHoursTemporarilyClosed'
+  }
+  if (isCurrentlyOpened) {
+    return 'openingHoursOpened'
+  }
+  return 'openingHoursClosed'
+}
+
+const OpeningHours = ({
+  isCurrentlyOpened,
+  direction,
+  openingHours,
+  isTemporarilyClosed,
+}: OpeningHoursProps): ReactElement | null => {
+  const { t } = useTranslation('pois')
+  moment.locale('de')
```

Comment:
Because we getting the date in an array with the german weekday order `Monday-Sunday` this has to be fixed. If not we mixing the days f.e. first array entry would be sunday. We don't get weekday information from cms

Reply:
Yes, just add a `@ts-expect-error`, it works. Moment should be removed soonish anyway:
``` ts
  // The opening hours loaded from the cms are ordered according to the german weekday order
  // @ts-expect-error Interface has wrong type, supplying true is necessary and works
  const weekdays = moment.localeData('de').weekdays(true)
  console.log(weekdays, moment.weekdays())
```